README.rst

```diff
@@ -11,7 +11,7 @@ ServiceX REST Server
 This is a flask app for receiving user requests and orchestrating the ServiceX
 services:
 
-.. image:: doc/sequence_diagram.png
+.. image:: https://servicex.readthedocs.io/en/latest/img/ServiceX-request-lifecycle.png
```

Comment:
What's the advantages of a direct link?  I understand that it gets rid of a layer of indirection, but I think that maybe we want that in case we change where the RtD images come from?

Reply:
The RtD link relies on Read the Docs being up, the success of our latest build, the repo RtD is linked to, and the path to the image in that repo.

The direct link relies only on the repo and path, RtD. If we were to point RtD at another repo (theoretically - not likely), that repo would need to have the same exact path as the current repo in order for this link to continue working. So an indirect link via RtD is effectively tightly coupled with the repo anyway, since the path ends up in the URL, but adds an extra dependency on RtD.

Reply:
Okay, is the link you provided correct?  Going to https://github.com/ssl-hep/ServiceX/blob/develop/docs/img/ServiceX-request-lifecycle.png  in my browser or httpie gives an html page.  How do I get a raw link?

Reply:
@sthapa - yes, please move this to a separate PR - it doesn't have anything to do with picking up config from environment vars, right?

Reply:
No, it's a separate issue.  I'll move this to another PR